Add skeleton file for LA

SnowCatSci opened the pull request:
Suggestions for where to eat and what to do

SnowCatSci committed:
Add skeleton file for LA

Suggestions for where to eat and what to do

mvcarran reviewed (changes requested):
hi can you break down LA by neighborhoods (los feliz, downtown...)?

chinampa commented:
nifty

SnowCatSci committed:
adding places to eat and see in Los Angeles

Added places to eat and see in Santa Monica, coastal neighborhood in greater LA area.

SnowCatSci commented:
@mvcarran Great point. There are many LA neighborhoods, some formal, some informal: Los Feliz, Highland Park, Korea Town, Central LA, Pasadena, Culver City, Santa Monica, Mid-city, Hollywood, West Hollywood, etc
